trace.go

```diff
@@ -12,20 +12,37 @@ type Tracer interface {
 
 type Span interface {
 	End()
+	AddStringAttribute(name, value string)
```

Comment:
Maybe just expose `AddAttributes()` here, so we don't have to declare our own span struct?

Reply:
If I only expose AddAttribute, then you will have to import the opencensus tracing package in order to call create attribute.

I was thinking that this was prefered over the other solution?

Reply:
Ok, that makes sense, then we keep it this way!